app/src/components/core/Anotacao.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import styled from "styled-components";
+
+import { Colors, Typography } from "../../config";
+import getSpacing from '../../config/spacing';
+import fontsSize from '../../config/fontsSize.js';
+
+export const Anotacao = styled.TouchableOpacity`
+ border-radius: 10px;
+ width: ${({tileSize2}) => tileSize2}px;
+ height: ${({tileSize}) => tileSize}px;
+ align-self: flex-start;
+ position: relative;
+ margin-top: ${getSpacing(30)};
```

Comment:
A margem entre blocos no Figma era de 16 unidades.
```suggestion
 margin-top: ${getSpacing(16)};
```